Display/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+    "log"
+    "net/http"
+    "time"
+
+    "github.com/gorilla/websocket"
+    zmq "github.com/pebbe/zmq4"
+)
+
+const (
+    writeWait = 10 * time.Second
+    pongWait = 60 * time.Second
+    pingPeriod = (pongWait * 9) / 10
+    maxMessageSize = 512
+    publisherSource = "tcp://localhost:5000"
+)
+
+var upgrader = websocket.Upgrader{}
+
+type subscription struct {
+    subscribe string
+}
+
+type Client struct {
+    conn *websocket.Conn
+}
+
+func (client *Client) readPump() {
+    defer func() {
+        log.Printf("read pump closing")
```

Comment:
Logging is a big thing in python: https://docs.python.org/3/library/logging.html

If you're going to be doing logging, make that a separate PR, I think.